python/smashbox/multiprocessing_engine.py

```diff
@@ -230,7 +241,9 @@ def add_worker(f,name=None):
     _smash_.parser = smashbox.compatibility.argparse.ArgumentParser()
     _smash_.parser.add_argument('test_target')
     _smash_.parser.add_argument('config_blob')
-
+    _smash_.parser.add_argument('loop_i')
+    _smash_.parser.add_argument('testset_i')
+    _smash_.parser.add_argument('test_doc')
     _smash_.args = _smash_.parser.parse_args()
 
```

Comment:
adding loop_i, testset_i and test_doc to config_blob will result in reduction of this lines